API.md

```diff
@@ -0,0 +1,1246 @@
+
+**Properties**
+
+-   `id` **[string][58]?** 
+-   `domain` **[string][58]?** 
+-   `external_id` **[string][58]?** 
+
+### THullAccount
+
+Account object with ident information and traits
+
+Type: {id: [string][58]}
+
+**Properties**
+
+-   `id` **[string][58]** 
+
+### THullAttributeName
+
+Attributes (also called traits) names are strings
+
+Type: [string][58]
+
+### THullAttributeValue
+
+Possible attribute (trait) values
+
+Type: ([string][58] \| [boolean][60] \| [Date][72] \| [Array][64]&lt;[string][58]>)
+
+### THullAttributesChanges
+
+Attributes (traits) changes is an object map where keys are attribute (trait) names and value is an array
+where first element is an old value and second element is the new value.
+This object contain information about changes on one or multiple attributes (that's thy attributes and changes are plural).
+
+Type: {}
+
+### THullConnector
+
+Connector (also called ship) object with settings, private settings and manifest.json
+
+Type: {id: [string][58], updated_at: [string][58], created_at: [string][58], name: [string][58], description: [string][58], tags: [Array][64]&lt;[string][58]>, manifest: [Object][57], settings: [Object][57], private_settings: [Object][57], status: [Object][57]}
+
+**Properties**
+
+-   `id` **[string][58]** 
+-   `updated_at` **[string][58]** 
+-   `created_at` **[string][58]** 
+-   `name` **[string][58]** 
+-   `description` **[string][58]** 
+-   `tags` **[Array][64]&lt;[string][58]>** 
+-   `manifest` **[Object][57]** 
+-   `settings` **[Object][57]** 
+-   `private_settings` **[Object][57]** 
+-   `status` **[Object][57]** 
+
+### THullEvent
+
+Hull Event object
+
+Type: {id: [string][58], event: [string][58], context: [Object][57], properties: [Object][57]}
+
+**Properties**
+
+-   `id` **[string][58]** 
+-   `event` **[string][58]** 
+-   `context` **[Object][57]** 
+-   `properties` **[Object][57]** 
+
+### THullObjectAttributes
+
+Object which is passed to `hullClient.asAccount().traits(traits: THullObjectAttributes)` call
+
+Type: (THullUserAttributes | THullAccountAttributes)
+
+### THullObjectIdent
+
+General type for THullUserIdent and THullAccountIdent
+
+Type: (THullUserIdent | THullAccountIdent)
+
+### THullObject
+
+General type for THullUser and THullAccount
+
+Type: (THullUser | THullAccount)
+
+### THullReqContext
+
+Context added to the express app request by hull-node connector sdk.
+Accessible via `req.hull` param.
+
+Type: {requestId: [string][58], config: [Object][57], token: [string][58], client: [Object][57], ship: THullConnector, connector: THullConnector, hostname: [string][58], options: [Object][57], connectorConfig: [Object][57], segments: [Array][64]&lt;THullSegment>, cache: [Object][57], metric: [Object][57], enqueue: [Function][61], helpers: [Object][57], service: [Object][57], shipApp: [Object][57], message: [Object][57]?, notification: [Object][57], smartNotifierResponse: [Object][57]?}
+
+**Properties**
+
+-   `requestId` **[string][58]** 
```

Comment:
Is it possible to render it as raw JS ?

I feel here a format such as 

```js
type {
  requestId: string
}
```

would be much more explicit